src/class/objects.php

```diff
@@ -1518,5 +1530,31 @@ function viewList ($sViewListID = false, $aColsToSkip = array(), $bNoHistory = f
 
         return $nTotal;
     }
+
+    public static function lovd_getObjectLinksHTML($aIDs, $urlFormat)
+    {
```

Comment:
Oops, I forgot to tell you... functions and methods are sorted alphabetically... could you sort this function to its correct location in the file? Also, methods are separated by 5 new lines.